--- src/sj-prefs.c	30 Jun 2005 19:42:13 -0000	1.43
+++ src/sj-prefs.c	8 Oct 2005 07:25:21 -0000
@@ -241,7 +241,7 @@
    * prefs window from the main is ugly. Maybe put the validation code
    * into sj-utils?
    */
-  const char* base_uri;
+  const char* base_uri, *uri;
   g_return_if_fail (strcmp (entry->key, GCONF_BASEURI) == 0);
 
   if (entry->value == NULL) {
@@ -249,7 +249,8 @@
   } else {
     g_return_if_fail (entry->value->type == GCONF_VALUE_STRING);
     base_uri = gconf_value_get_string (entry->value);
-    if (strcmp (gtk_file_chooser_get_current_folder_uri (GTK_FILE_CHOOSER (basepath_fcb)), base_uri) != 0) { 
+    uri = gtk_file_chooser_get_current_folder_uri (GTK_FILE_CHOOSER (basepath_fcb));
+    if (uri == NULL || strcmp (uri, base_uri) != 0) { 
       gtk_file_chooser_set_current_folder_uri (GTK_FILE_CHOOSER (basepath_fcb), base_uri);
     }
   }
