diff -urN sound-juicer-2.16.2/data/sound-juicer.desktop.in.in sound-juicer-2.16.2.new/data/sound-juicer.desktop.in.in
--- sound-juicer-2.16.2/data/sound-juicer.desktop.in.in	2006-10-17 22:58:10.000000000 +0200
+++ sound-juicer-2.16.2.new/data/sound-juicer.desktop.in.in	2006-11-20 15:02:54.000000000 +0100
@@ -8,7 +8,7 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;AudioVideo;
+Categories=GTK;GNOME;AudioVideo;Audio;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=sound-juicer
 X-GNOME-Bugzilla-Component=general
