diff -aurN sound-juicer-0.5.13.orig/src/cd-drive.c sound-juicer-0.5.13/src/cd-drive.c
--- sound-juicer-0.5.13.orig/src/cd-drive.c	2004-09-21 14:57:28.000000000 +0200
+++ sound-juicer-0.5.13/src/cd-drive.c	2004-09-28 16:39:27.000000000 +0200
@@ -351,16 +351,8 @@
 		int num_devices;
 		CDMediaType type;
 		char *hal_type;
-		LibHalFunctions hal_functions = {
-			NULL, /* mainloop integration */
-			NULL, /* device_added */
-			NULL, /* device_removed */
-			NULL, /* device_new_capability */
-			NULL, /* property_modified */
-			NULL, /* device_condition */
-		};
 		
-		ctx = hal_initialize (&hal_functions, FALSE);
+		ctx = hal_initialize (NULL, FALSE);
 		if (ctx != NULL) {
 			device_names = hal_manager_find_device_string_match (ctx, 
 									     "info.parent",
@@ -485,20 +477,12 @@
 hal_scan (gboolean recorder_only)
 {
 	GList *cdroms = NULL;
-	LibHalFunctions hal_functions = {
-		NULL, /* mainloop integration */
-		NULL, /* device_added */
-		NULL, /* device_removed */
-		NULL, /* device_new_capability */
-		NULL, /* property_modified */
-		NULL, /* device_condition */
-	};
 	int i;
 	int num_devices;
 	char** device_names;
 	LibHalContext *ctx;
 
-	ctx = hal_initialize (&hal_functions, FALSE);
+	ctx = hal_initialize (NULL, FALSE);
 	if (ctx == NULL) {
 		return NULL;
 	}
diff -aurN sound-juicer-0.5.13.orig/src/Makefile.am sound-juicer-0.5.13/src/Makefile.am
--- sound-juicer-0.5.13.orig/src/Makefile.am	2004-09-08 16:51:04.000000000 +0200
+++ sound-juicer-0.5.13/src/Makefile.am	2004-09-28 16:39:27.000000000 +0200
@@ -13,7 +13,7 @@
 	-DPIXMAPDIR=\""$(datadir)/pixmaps"\" \
 	-DPKGDATADIR=\""$(datadir)/sound-juicer"\" \
 	@MUSICBRAINZ_CFLAGS@ @GSTREAMER_CFLAGS@ \
-	@UI_CFLAGS@
+	@UI_CFLAGS@ @HAL_CFLAGS@
 
 
 # We always build Sound Juicer
@@ -45,7 +45,7 @@
 	$(BACONFILES) \
 	$(MARSHALFILES)
 
-sound_juicer_LDFLAGS = @MUSICBRAINZ_LIBS@ @GSTREAMER_LIBS@ @UI_LIBS@
+sound_juicer_LDFLAGS = @MUSICBRAINZ_LIBS@ @GSTREAMER_LIBS@ @UI_LIBS@ @HAL_LIBS@
 
 
 #
diff -aurN sound-juicer-0.5.13.orig/tests/Makefile.am sound-juicer-0.5.13/tests/Makefile.am
--- sound-juicer-0.5.13.orig/tests/Makefile.am	2004-06-10 16:35:21.000000000 +0200
+++ sound-juicer-0.5.13/tests/Makefile.am	2004-09-28 16:39:27.000000000 +0200
@@ -5,7 +5,7 @@
 	-DG_DISABLE_DEPRECATED=1 \
 	-I$(top_srcdir)/src \
 	@MUSICBRAINZ_CFLAGS@ @GSTREAMER_CFLAGS@ \
-	@UI_CFLAGS@
+	@UI_CFLAGS@ @HAL_CFLAGS@
 
 AM_CFLAGS = -g -Wall -export-dynamic -Wunused
 
@@ -21,4 +21,4 @@
 	$(top_srcdir)/src/transport.hxx \
 	$(top_srcdir)/src/dvd_plus_rw_utils.cpp
 
-mb_test_LDADD = @UI_LIBS@ @MUSICBRAINZ_LIBS@
+mb_test_LDADD = @UI_LIBS@ @MUSICBRAINZ_LIBS@ @HAL_LIBS@
