--- sound-juicer-2.32.0/src/sj-main.c	2010-08-04 06:35:35.000000000 -0700
+++ sound-juicer-2.32.0/src/sj-main.c.new	2010-11-12 16:41:26.401868408 -0800
@@ -49,6 +49,11 @@
 #include "sj-play.h"
 #include "sj-genres.h"
 
+/* gtk compatibility hack, remove when required gtk >= 2.91.0 */
+#if (!GTK_CHECK_VERSION(2,24,0) || (GTK_CHECK_VERSION(2,90,0) && !GTK_CHECK_VERSION(2,91,0)))
+#define gdk_window_get_display gdk_drawable_get_display
+#endif
+
 gboolean on_delete_event (GtkWidget *widget, GdkEvent *event, gpointer user_data);
 
 static void reread_cd (gboolean ignore_no_media);
@@ -886,10 +891,10 @@
 
   /* Set watch cursor */
   if (realized) {
-    cursor = gdk_cursor_new_for_display (gdk_drawable_get_display (window), GDK_WATCH);
+    cursor = gdk_cursor_new_for_display (gdk_window_get_display (window), GDK_WATCH);
     gdk_window_set_cursor (window, cursor);
     gdk_cursor_unref (cursor);
-    gdk_display_sync (gdk_drawable_get_display (window));
+    gdk_display_sync (gdk_window_get_display (window));
   }
 
   /* Set statusbar message */
