diff -urN aa/sound-juicer-0.5.10.1/configure.in sound-juicer-0.5.10.1/configure.in
--- aa/sound-juicer-0.5.10.1/configure.in	2004-02-05 17:40:43.000000000 +0100
+++ sound-juicer-0.5.10.1/configure.in	2004-02-20 18:16:29.375731968 +0100
@@ -14,7 +14,7 @@
 GETTEXT_PACKAGE=sound-juicer
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [The name of the gettext package.])
 AC_SUBST(GETTEXT_PACKAGE)
-ALL_LINGUAS="ar az be bg ca cs da de el en_CA en_GB es eu fi fr hr hu it ja ko lt ml ms nl no pl pt pt_BR sr sr@Latn sv th uk zh_CN zh_TW"
+ALL_LINGUAS="ar az be bg ca cs da de el en_CA en_GB es eu fi fr hr hu it ja ko lt ml ms nb nl pl pt pt_BR sr sr@Latn sv th uk zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 
