diff -urN aa/sound-juicer-0.5.10.1/configure.in sound-juicer-0.5.10.1/configure.in
--- aa/sound-juicer-0.5.10.1/configure.in	2004-02-05 17:40:43.000000000 +0100
+++ sound-juicer-0.5.10.1/configure.in	2004-03-10 20:33:28.839184264 +0100
@@ -51,11 +51,19 @@
 # Find GStreamer.
 HAVE_GSTREAMER=no
                                                                                 
-# Look for 0.7
+# Look for 0.8
+GST_MAJORMINOR=0.8
+PKG_CHECK_MODULES(GSTREAMER, gstreamer-$GST_MAJORMINOR,
+        HAVE_GSTREAMER=yes,
+        HAVE_GSTREAMER=no)
+
+# Then look for 0.7
+if test "x$HAVE_GSTREAMER" = "xno"; then
 GST_MAJORMINOR=0.7
 PKG_CHECK_MODULES(GSTREAMER, gstreamer-$GST_MAJORMINOR,
         HAVE_GSTREAMER=yes,
         HAVE_GSTREAMER=no)
+fi
                                                                                 
 # No? Try 0.6
 if test "x$HAVE_GSTREAMER" = "xno"; then
