--- sound-juicer-2.32.0/configure.in	2010-09-28 02:54:47.000000000 -0700
+++ sound-juicer-2.32.0/configure.in.new	2010-11-12 17:56:17.305920795 -0800
@@ -55,7 +55,7 @@
        GTK_REQUIRED=2.90.0
        LIBBRASERO_MEDIA_PC=libbrasero-media3
        LIBCANBERRA_GTK_PC=libcanberra-gtk3
-       GNOME_MEDIA_PROFILES_PC=gnome-media-profiles-3.0
+       GNOME_MEDIA_PROFILES_PC=libgnome-media-profiles-3.0
        ;;
 esac
 
--- sound-juicer-2.32.0/libjuicer/sj-extractor.c	2010-02-09 05:33:28.000000000 -0800
+++ sound-juicer-2.32.0/libjuicer/sj-extractor.c.new	2010-11-12 18:37:07.622546893 -0800
@@ -30,7 +30,7 @@
 #include <glib-object.h>
 #include <gst/gst.h>
 #include <gst/tag/tag.h>
-#include <profiles/gnome-media-profiles.h>
+#include <libgnome-media-profiles/gnome-media-profiles.h>
 #include "sj-extractor.h"
 #include "sj-structures.h"
 #include "sj-error.h"
--- sound-juicer-2.32.0/src/sj-main.c	2010-08-04 06:35:35.000000000 -0700
+++ sound-juicer-2.32.0/src/sj-main.c.new2	2010-11-12 18:40:21.779342318 -0800
@@ -33,7 +33,7 @@
 #include <gconf/gconf-client.h>
 #include <brasero-medium-selection.h>
 #include <brasero-volume.h>
-#include <profiles/gnome-media-profiles.h>
+#include <libgnome-media-profiles/gnome-media-profiles.h>
 #include <gst/gst.h>
 
 #include "bacon-message-connection.h"
--- sound-juicer-2.32.0/src/sj-prefs.c	2010-05-26 15:53:08.000000000 -0700
+++ sound-juicer-2.32.0/src/sj-prefs.c.new	2010-11-12 18:40:30.304672876 -0800
@@ -25,7 +25,7 @@
 #include <string.h>
 #include <gtk/gtk.h>
 #include <gconf/gconf-client.h>
-#include <profiles/gnome-media-profiles.h>
+#include <libgnome-media-profiles/gnome-media-profiles.h>
 #include <brasero-drive-selection.h>
 
 #include "sj-util.h"
--- sound-juicer-2.32.0/libjuicer/sj-extractor.h	2010-02-09 05:33:28.000000000 -0800
+++ sound-juicer-2.32.0/libjuicer/sj-extractor.h.new	2010-11-12 18:51:04.945437028 -0800
@@ -26,7 +26,7 @@
 #include <glib.h>
 #include <glib-object.h>
 #include <gio/gio.h>
-#include <profiles/audio-profile.h>
+#include <libgnome-media-profiles/audio-profile.h>
 #include "sj-structures.h"
 
 G_BEGIN_DECLS
