diff -urN sound-juicer-0.6.0.orig/data/sound-juicer.desktop.in sound-juicer-0.6.0/data/sound-juicer.desktop.in
--- sound-juicer-0.6.0.orig/data/sound-juicer.desktop.in	2005-02-11 00:09:09.839882912 +0100
+++ sound-juicer-0.6.0/data/sound-juicer.desktop.in	2005-02-11 00:10:36.894648576 +0100
@@ -7,7 +7,7 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GNOME;Application;AudioVideo;
+Categories=GTK;GNOME;Audio;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=sound-juicer
 X-GNOME-Bugzilla-Component=general
